scripts/backup-aliases.sh

```diff
@@ -5,7 +5,7 @@ cd ${ARCHIVEHOME}
 git fetch origin
 git pull origin master
 
-scp -r chris@penguin.gtalug.org:/etc/aliases ${ARCHIVEHOME}
+scp -r ${USER}@penguin.gtalug.org:/etc/aliases ${ARCHIVEHOME}
```

Comment:
Yes, maybe in the next pull request, but for now more generic code is better than less generic code.